react-basic-practice/src/helpers/validate.js

```diff
@@ -0,0 +1,33 @@
+import REGEXP from '../constants/regexp';
+
+const Validate = (dataInput) => {
```

Comment:
Function name should be camelCase. Also what are you validate here?

Reply:
This function is to validate the form input. I will change the name to `validateInput`